web/src/main/java/org/openmrs/web/filter/util/FilterUtil.java

```diff
@@ -28,6 +28,14 @@
  */
 public class FilterUtil {
 	
+	/*
+	Utility classes should not have public constructors.
+        Utility classes, which are a collection of static members, are not meant to be instantiated.
+	*/
+	
+	private FilterUtil() {
+        }
```

Comment:
The alignment here is out. Isn't it?

Reply:
Removed the unrequired whitespace sir !!